src/machine-learning/machine-learning.ts

```diff
@@ -109,8 +110,13 @@ export class MachineLearning implements FirebaseServiceInterface {
    * @return {Promise<Model>} A Promise fulfilled with the updated model.
    */
   public updateModel(modelId: string, model: ModelOptions): Promise<Model> {
-    throw new Error('NotImplemented');
-  }
+     const updateMask = getMaskFromOptions(model);
+     return this.maybeSignUrl(model, true)
+      .then((modelContent) => this.client.updateModel(modelId, modelContent, updateMask))
+      .then((operation) => {
```

Comment:
Let's be consistent. Either use the one-liner format or use the block format.

```
.then((x) => doStuff(x))
```

or 

```
.then((x) => {
  return doStuff(x);
})
```

I personally prefer the latter. But let's use one format throughout the file.

Reply:
Oops, sorry I had some logging in there for debugging and forgot to put it back.